types/snarkjs/index.d.cts

```diff
@@ -0,0 +1,118 @@
+export as namespace snarkjs;
+
+export namespace fflonk {
+    function exportSolidityCallData(_pub: any, _proof: any): Promise<any>;
+    function exportSolidityVerifier(vk: any, templates: any, logger?: any): Promise<any>;
+    function fullProve(_input: any, wasmFilename: any, zkeyFilename: any, logger?: any): Promise<any>;
+    function prove(zkeyFileName: any, witnessFileName: any, logger?: any): Promise<any>;
+    function setup(r1csFilename: any, ptauFilename: any, zkeyFilename: any, logger?: any): Promise<any>;
+    function verify(_vk_verifier: any, _publicSignals: any, _proof: any, logger?: any): Promise<boolean>;
+}
+
+export namespace groth16 {
+    function exportSolidityCallData(_proof: any, _pub: any): Promise<any>;
+    function fullProve(_input: any, wasmFile: any, zkeyFileName: any, logger?: any): Promise<any>;
```

Comment:
In my project I created the following signature:
```typescript
export function fullProve<InputSignals extends CircuitSignals>(_input: InputSignals, wasmFilename: string, zkeyFilename: string, logger?: any): Promise<{ proof: object, publicSignals: string[] }>

type CircuitSignals<T extends readonly string[] = []> = T extends []
  ? {[signal: string]: SignalValueType}
  : {[signal in T[number]]: SignalValueType};

type IntegerValueType = `${number}` | number | bigint;

type SignalValueType = IntegerValueType | SignalValueType[];
```

This is taken from https://github.com/erhant/circomkit/blob/main/src/testers/proofTester.ts